public/main.css

```diff
@@ -40,6 +33,22 @@ h1 {
     box-shadow: inset 0 0 10px #000000;
 }
 
+.filter-button {
+    background-color: rgba(140, 255, 83, 0.87);
+    color: #9b32ff;
+    margin-left: 5px;
+    margin-bottom: 5px;
+    border-radius: 10px;
+    box-shadow: inset 0 0 10px #000000;
+}
+
+.incomplete-counter {
+    align-content: center;
```

Comment:
`align-content` should only be used with flex. Are you trying to center the content within the div or center the div itself?